sources/packages/backend/apps/api/src/route-controllers/student-loan-balance/_tests_/e2e/student-loan-balance.aest.controller.getStudentLoanBalance.e2e-spec.ts

```diff
@@ -0,0 +1,149 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import { MAXIMUM_ESSENTIAL_LOAN_BALANCE_RECORDS } from "../../../../utilities";
+
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import { createFakeStudentLoanBalance } from "@sims/test-utils/factories/student-loan-balance";
+import { addToDateOnlyString } from "@sims/utilities";
+
+describe("StudentLoanBalanceAESTController(e2e)-getStudentLoanBalance", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it(
+    "Should get all the student loan balance records," +
```

Comment:
No need for a comma and I believe that the other E2E descriptions do not have it.